src/app/components/AddressBox/__tests__/index.tsx

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react'
+import copy from 'copy-to-clipboard'
+import { screen } from '@testing-library/dom'
+import { render } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+
+import { AddressBox } from '../index'
+
+jest.mock('copy-to-clipboard')
+
+const testAddress = 'oasis1qqurxkgavtcjjytneumeclx59ds3avjaqg7ftqph'
+
+const renderComponent = () => render(<AddressBox address={testAddress} />)
+
+describe('<AddressBox />', () => {
+  it('should render address properly', () => {
+    const { container } = renderComponent()
+    expect(container).toMatchSnapshot()
+  })
+
+  it('should be able to copy address to clipboard', async () => {
+    renderComponent()
+    await userEvent.click(screen.getByTestId('copy-address'))
+    expect(copy).toHaveBeenCalledWith(testAddress)
+  })
```

Comment:
Actually I wanted to test both the actual clipboard copy _and_ the notification, since AddressBox had no tests before.
Now I have added a new test case below, specifically for the notification.